e2e/service_cometbft_test.go

```diff
@@ -0,0 +1,69 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"net/http"
+
+	vegeta "github.com/tsenart/vegeta/lib"
+
+	"github.com/buildwithgrove/path/qos/jsonrpc"
+)
+
+/* -------------------- CometBFT REST Endpoint Definitions -------------------- */
+
+var cometbftExpectedID = jsonrpc.IDFromInt(-1)
+
+// Reference for all CometBFT RPC endpoints:
+// - https://docs.cometbft.com/v0.38/rpc/
+const (
+	cometbftEndpointStatus         = "/status"          // Get node status
+	cometbftEndpointHealth         = "/health"          // Get node health
+	cometbftEndpointNetInfo        = "/net_info"        // Get network info
+	cometbftEndpointConsensusState = "/consensus_state" // Get consensus state
+	cometbftEndpointCommit         = "/commit"          // Get commit
+	cometbftEndpointABCIInfo       = "/abci_info"       // Get ABCI info
+	cometbftEndpointBlock          = "/block"           // Get block at height
+	cometbftEndpointBlockResults   = "/block_results"   // Get block results
+	cometbftEndpointValidators     = "/validators"      // Get validators
+)
+
+// getCometBFTTestEndpoints returns all CometBFT endpoints for a service load test.
+func getCometBFTTestEndpoints() []string {
```

Comment:
Endpoint is short for `Service Provider Endpoint` in this repo.
Let's not overload the name. Something like URL path, etc. will be less confusing.

Reply:
Done, updated to URL paths.